### PR TITLE
openvswitch: 3.1.1 -> 3.2.1, openvswitch-lts: 2.17.6 -> 2.17.8

### PR DESCRIPTION
**Pull request by adamcstephens**
## Description of changes

openvswitch: add updateScript
openvswitch-lts: 2.17.6 -> 2.17.8
openvswitch: 3.1.1 -> 3.2.1

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by adamcstephens** [comment]
Result of `nixpkgs-review pr 274126` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>openvswitch</li>
    <li>openvswitch-lts</li>
    <li>openvswitch-lts.man</li>
    <li>openvswitch.man</li>
    <li>pipework</li>
  </ul>
</details>

-----

Result of `nixpkgs-review pr 274126` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages failed to build:</summary>
  <ul>
    <li>openvswitch</li>
    <li>openvswitch-lts</li>
    <li>openvswitch-lts.man</li>
    <li>openvswitch.man</li>
    <li>pipework</li>
  </ul>
</details>

File: pkgs/os-specific/linux/openvswitch/update.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i nu -p common-updater-scripts
```
Comment (mweinelt) [inline pkgs/os-specific/linux/openvswitch/update.sh:2]:
```suggestion
#!nix-shell -i nu -p nushell common-updater-scripts
```

Not everyone has nushell already installed!

**Review by mweinelt — CHANGES_REQUESTED** [review]
Make the updateScript only update the derivation it was called for. Otherwise loving that we get an update script after all that time!

**Comment by mweinelt** [comment]
Result of `nixpkgs-review pr 274126` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>10 packages marked as broken and skipped:</summary>
  <ul>
    <li>libvmi</li>
    <li>qemu_xen</li>
    <li>qemu_xen-light</li>
    <li>qemu_xen-light.ga</li>
    <li>qemu_xen.ga</li>
    <li>qemu_xen_4_15</li>
    <li>qemu_xen_4_15-light</li>
    <li>qemu_xen_4_15-light.ga</li>
    <li>qemu_xen_4_15.ga</li>
    <li>qubes-core-vchan-xen</li>
  </ul>
</details>
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>openvswitch</li>
    <li>openvswitch-lts</li>
    <li>openvswitch-lts.man</li>
    <li>openvswitch.man</li>
    <li>pipework</li>
  </ul>
</details>

File: pkgs/os-specific/linux/openvswitch/update.sh (hunk 1 above)
Comment (adamcstephens) [reply pkgs/os-specific/linux/openvswitch/update.sh]: Oops, I had nushell in originally as the interpreter, but that's a command not a package and I never added it to the package list. Fixed.

**Review by mweinelt — APPROVED** [review]

**Pull request merged**